draft-ietf-taps-arch.md

```diff
@@ -524,7 +532,7 @@ The following categories of events can be delivered to an application:
 
 ### Connection Groups
 
-A Connection Group is a set of Connections that shares Connection Properties and cached state generated by protocols.
+A Connection Group is a set of Connections that shares Connection Properties and Cached State generated by protocols.
```

Comment:
I don't think we have a cached state objet here … so why capitalize?

Reply:
reverted - good either way